src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -86,14 +91,21 @@ private bool ShouldIncludeInTargetTypedCompletionList(ISymbol symbol, ImmutableA
                 return false;
             }
 
+            if (typeConvertibilityCache.TryGetValue(type, out var isConvertible))
+            {
+                return isConvertible;
+            }
+
             foreach (var inferredType in inferredTypes)
             {
                 if (semanticModel.Compilation.ClassifyCommonConversion(type.WithoutNullability(), inferredType.WithoutNullability()).IsImplicit)
```

Comment:
Should we do this once before the loop? `type.WithoutNullability()`

Reply:
Addressed. Thanks @JoeRobich!